content/en/References/_index.adoc

```diff
@@ -11,3 +11,4 @@ description: >
 * https://docs.google.com/document/d/164DD9j-OUmsBKB3FELcanB-5L1GkFy0qX10EiUfEtjM/edit[Testing Questions]
 * https://docs.google.com/spreadsheets/d/1FWTYRxglWDqIK_xI6HlBGKUegICQRsXe5NHCJTlTlHs/edit?usp=sharing[Printing Queue]
 * https://docs.google.com/presentation/d/1UeSRyUB0dAOZZecX4TGh40GkVtGADlXzBOwVFjPqIFE/edit#slide=id.g7f621674bc_0_63[System Concept Diagram]
+* https://drive.google.com/file/d/1lFtztYyWjZQqkgim3zkKvQTohYCQVQrx/view?usp=sharing[Circuit Board UML Diagram]
```

Comment:
I would remove this from google drive, and instead add a dedicated adoc file in the same directory with the circuit schematic plantuml as an embedded image using the asciidoctor syntax plantuml (see example here https://github.com/303adastra/pid-controlled-clock-drive/blame/deploy/content/References/reference-architecture.adoc#L13). Reason for this being the whole point of migrating to plantuml is to be able to generate these diagrams from plaintext.